assets/images/cover-autoyast.svg

```diff
@@ -0,0 +1 @@
.8 3.639-5.618 6.55-9.359 9.5l-15.32 11.962c3.05 1.481 5.783 3.422 8.298 5.72-2.706-1.494-5.336-3.064-8.49-3.894-2.196-.345-4.366-.32-6.37 1.277-1.801 1.493-1.597 4.021-1.227 6.383.69 3.652 3.741 6.895 7.214 9.346 3.23 1.443 6.205.702 9.065-.983 2.477-2.298 3.485-6.333 3.179-11.848 1.787-.332 1.391 10.303 1.71 9.09.167-.651 26.263-13.7 33.17-15.32 7.048-1.814 18.04-2.401 25.994-4.878 4.941-1.583 8.555-5.337 11.619-10.048l-2.464 5.235a14.082 14.082 0 01-8.095 6.128l-4.43 47.137-.09 45.58-7.711 11.49 5.247-.179 5.592-7.392-4.11 8.848s26.9-1.8 26.9-1.609c0 .192-8.299-54.516-8.299-54.516s8.989-10.048 9.18-10.189c.192-.153 32.876 11.031 32.876 11.031l-5.03-5.937-4.137-2.285 5.017 1.66s11.657 14.12 11.708 14.325c.051.191 2.081-2.54 2.081-2.54l-12.205 49.536 16.214 5.528-.51-7.57 3.88 9.396 5.963.396-4.034-9.793 7.022 8.529 3.855-.766-6.945-9.997 8.107-48.26-9.346-7.38-7.239-9.703 6.856 6.984-7.43-10.584-.587-13.98 2.604 12.04 2.809-9.896-1.456 11.555 5.401 11.848 10.801 8.043-.204 1.852 8.503-3.116 5.49-12.41.472-7.685.792 7.788s-4.724 11.44-4.724 11.835c0 .383 6.167-5.835 6.167-5.835l-6.767 7.967 10.814 19.509 3.664 21.27 2.694-1.928-.575.766-8.681 11.337 5.694-1.238 2.209-3.498-.97 4.264 2.03.102-.473 4.66 4.239-3.792 3.166.983.983-5.656v5.746l2.975-.588 1.442 4.852 2.12-6.116 4.15-.191c-3.959-30.029-9.193-60.057-10.598-90.086-.153-5.72-4.953-15.832-10.124-22.445-6.04-6.958-18.372-21.832-35.876-26.007-9.55-1.136-19.279.408-29.876 3.868-9.371 2.745-19.049 6.09-29.569 11.133 2.477 4.06 5.758 8.044 11.35 12.768 1.75 1.71 4.903 2.183 8.005 1.94-2.145.766-4.353 1.405-6.664.192-3.767-2.554-6.869-5.745-9.831-9.116a35.25 35.25 0 01-4.532-7.29c-1.788-4.06.319-9.218 3.766-14.76 3.128-3.842 6.575-7.596 13.584-10.29 6.307-2.515 12.64-5.03 18.628-8.35 6.767-4.391 17.568-18.384 23.313-28.164-15.653 5.745-31.293 10.341-46.856 19.508-5.822 3.997-11.529 10.189-17.35 16.598-2.478 2.809-5.159 5.439-8.683 7.303-4.838 2.592-8.17.127-10.405-1.851a74.344 74.344 0 01-9.154-12.219c-3.447-4.724-7.06-9.46-12.346-14.184-6.077-5.643-11.555-12.155-17.07-18.066zm99.023 15.487L332.65 92.225l9.358-3.243-18.832 9.23zm-65.407 34.944c1.213-.026 2.426 0 3.6.128-7.621.587-11.107 1.902-17.72 5.298 2.425-1.851 3.792-3.23 8.158-4.724a30.695 30.695 0 015.962-.702zm2.12 10.52c3.766.064 6.32 1.724 6.32 1.724-8.35-2.247-12.934-.383-16.509 5.081-.42-.345 2.375-5.094 6.078-6.218a13.256 13.256 0 014.11-.587zm-.639 2.809a9.703 9.703 0 012.362.192c1.558.549 2.273 1.71 3.039 2.808-1.622-1.43-3.294-2.553-5.324-2.03 2.298.256 3.154 1.698 3.485 3.588.281 2.54-1.902 4.034-3.357 4.251-1.456.217-3.32-.153-4.392-1.723-.307-.639-1.124-2.988.178-4.66a12.767 12.767 0 00-3.855 2.451c.983-1.43 2.183-2.553 3.996-3.753a8.503 8.503 0 013.868-1.137zm-61.615 33.157c1.954 2.31 3.856 4.532 4.788 7.596.128 2.835-1.123 2.77-2.35 2.758-1.02-.013-2.961-1.175-2.336-2.12a7.482 7.482 0 00.741-3.957zm-7.15.702c1.533 1.609 3.04 2.681 3.32 4.928.115.536.217 1.864-.843 1.66-3.09-2.924-1.225-2.605-2.464-6.588z" fill="#fff"/></svg>
```

Comment:
These are cool! Thanks!

Did either of you come up with a favicon yet?

Reply:
>     * Could you update the `Makefile` instead of deleting it? I figure we'll still need it for uploading content & it doesn't hurt to have the whole `bundle exec ...` stuff not just written in a README but actually easily runnable from the Makefile.

Added

>     * How version-sensitive is this? I've used a Ruby-based HTML generation script about once so far and had to immediately go into this script and change it, because it wouldn't run, requesting some dependency version I didn't have installed.

Ruby version shouldn't matter. All you need to do is `sudo zypper in ruby ruby-devel && bundle exec jekyll build` .

>     * Where are the PO(T) files for Weblate popping out or can Weblate directly translate the YAMLs?

*.yaml files are well supported by Weblate.

>     * Please fix [openSUSE/chameleon#26](https://github.com/openSUSE/chameleon/issues/26) so we can get rid of the CSS quirk in this repo.

I cannot reproduce the issue. Can you provide a screenshot?